metrics-exporter-prometheus/tests/http_listener_integration_test.rs

```diff
@@ -36,7 +36,7 @@ mod http_listener_test {
             let labels = vec![Label::new("wutang", "forever")];
             let key = Key::from_parts("basic_gauge", labels);
             let gauge = recorder.register_gauge(&key, &METADATA);
-            gauge.set(-3.14);
+            gauge.set(-1.23);
```

Comment:
Clippy complained about this being too close to PI - easier to change it than to add an allow. Oddly, this was the only line it complained about (there are other similar places where this is used).